astropy/utils/console.py

```diff
@@ -386,6 +386,50 @@ def strip_ansi_codes(s):
     """
     return re.sub('\033\\[([0-9]+)(;[0-9]+)*m', '', s)
 
+def human_help(quantity,detail=False):
+    """
+    Return value in a human readable unit
+
+    Parameters
+    ----------
+    quantity:
+        Value to be converted to human readable unit.
+
+    detail:bool
+        choice of user to get list of all human readable units
+        or to get only the most appropriate unit.
+    """
+    from astropy import units as u
+    from astropy.units import get_current_unit_registry
+    ureg = get_current_unit_registry()
+    units = []
+    if quantity.si.unit==u.s and quantity.si.value>1 :
```

Comment:
I'd write `quantity.unit.is_equivalent(u.s) and quantity > 1*u.s`

More importantly: this will break if the quantity is not a scalar. Probably this is fine, but I would then catch non-scalar entries before even getting here (you can use `if not q.isscalar:`).